src/modules/infobox/components/FeatureProfileInformation.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div class="d-flex p-2 profile-popup-info-buttons">
+        <div
+            class="flex-grow-1 profile-info-container d-flex border border-light ps-1 pe-4 py-1"
+            data-cy="profile-popup-info-container"
+        >
+            <span
+                v-for="info in profileInformationFormat"
+                :key="info.title"
+                :title="$t(info.title)"
+                class="mx-2 text-nowrap d-flex align-content-center align-self-center"
+                :data-cy="`profile-popup-info-${info.title}`"
+            >
+                <font-awesome-icon
+                    v-for="(icon, index) in info.icons"
+                    :key="`${info.title}-${index}`"
+                    :icon="icon"
+                    class="me-1 align-self-center"
+                />
+                &nbsp;
+                <span data-cy="profile-popup-info">
+                    {{ info.value }}
+                </span>
+            </span>
+        </div>
+        <slot />
+    </div>
+</template>
+
+<script>
+import { GeoAdminProfile } from "@/api/profile.api";
+import { formatTime } from '@/modules/drawing/lib/drawingUtils'
+import { format } from '@/utils/numberUtils'
+
+export default {
+    props: {
+        profile: {
+            type: GeoAdminProfile,
```

Comment:
We have a Vue warning here because the profile is required but it can be null ! Either remove the required or allow for null value or when instanciating the FeatureProfile create an empty GeoadminProfile instead of null.

Reply:
good catch, thanks